aleo/caip10.md

```diff
@@ -2,7 +2,7 @@
 namespace-identifier: aleo-caip10
 title: Aleo Network - Namespace Accounts
 author: Jonathan Gonzalez (@jonandgon, jonathan@puzzle.online)
-discussions-to: <URL of PR, mailing list, etc>
+discussions-to: https://discord.com/invite/puzzlearcade
```

Comment:
this is the invite to a specific wallet? generally we prefer public resources to authenticated servers; maybe just link to https://github.com/ChainAgnostic/namespaces/pull/129 (i.e. here now) and include the Discord link in the github discussion thread instead?

Reply:
```suggestion
discussions-to: https://github.com/ChainAgnostic/namespaces/pull/129
```